ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -1140,7 +1169,10 @@ - (void)flipToNextPageFrom:(ORKStepViewController *)fromController {
         return;
     }
     
-    ORKStep *step = [self nextStep];
+    ORKStep *step = fromController.parentReviewStep;
+    if (!step) {
+        step = [self nextStep];
+    }
```

Comment:
I just tried the app again. Do you feel all the actions on step being reviewed should be `going backward`? Because the user is going back the the review step.